### PR TITLE
react and coffeescript example

### PR DESCRIPTION
**Pull request by kulicuu**
This is intended simply as a quick and dirty example of getting React working nicely with SocketStream.  There is nothing especially interesting about the app itself at this point -- that may or may not change in the next little while, time depending.

### CODE REVIEW

**Comment by kulicuu** [comment]
@arxpoetica 
After pulling and trying to merge from upstream I had a bunch of merge conflicts with the Ractive stuff-- my own fault.  I think I've set everything as you had it (actually manually copying raw from Github) -- let me know if any problems.  (thanks & sorry to have to check)

File: ractive-koa/client/code/libs/jquery.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...rea:[1,"<map>","</map>"],_default:[0,"",""]},bA=bk(e),bB=bA.appendChild(e.createElement("div"));bz.optgroup=bz.option,bz.tbody=bz.tfoot=bz.colgroup=bz.caption=bz.thead,bz.th=bz.td,p.support.htmlSerialize||(bz._default=[1,"X<div>","</div>"]),p.fn.extend({text:function(a){return p.access(this,function(a){return a===b?p.text(this):this.empty().append((this[0]&&this[0].ownerDocument||e).createTextNode(a))},null,a,arguments.length)},wrapAll:function(a){if(p.isFunction(a))return this.each(function(b){p(this).wrapAll(a.call(this,b))});if(this[0]){var b=p(a,this[0].ownerDocument).eq(0).clone(!0);this[0].parentNode&&b.insertBefore(this[0]),b.map(function(){var a=this;while(a.firstChild&&a.firstChild.nodeType===1)a=a.firstChild;return a}).append(this)}return this},wrapInner:function(a){return p.isFunction(a)?this.each(function(b){p(this).wrapInner(a.call(this,b))}):this.each(function(){var b=p(this),c=b.contents();c.length?c.wrapAll(a):b.append(a)})},wrap:function(a){var b=p.isFunction(a);return this.each(function(c){p(this).wrapAll(b?a.call(this,c):a)})},unwrap:function(){return this.parent().each(function(){p.nodeName(this,"body")||p(this).replaceWith(this.childNodes)}).end()},append:funct
```
Comment (arxpoetica) [inline ractive-koa/client/code/libs/jquery.min.js:2]: @kulicuu I removed jQuery from this folder.

**Comment by arxpoetica** [comment]
Yeah, @kulicuu I'm confused why there are differences here. Most of these changes are old, so I think you might have messed things up. Fortunately, I'm working on a separate branch which has all the correct changes (at least locally).

I'll make sure to back up my work before merging back again.

Any idea why this is messed up?

**Comment by kulicuu** [comment]
Yeah sorry I had a merge conflict when trying to pull your stuff (master) before redoing my pull request.  For some reason there were merge conflicts (I have no idea why as I never edited any of the Ractive files), and I apologise I just grabbed your files from the master branch to make sure -- wrote over.  You can merge again, and I'll pull again.  Hopefully should'nt be a problem anymore.

**Comment by arxpoetica** [comment]
I think I know why. Possibly you didn't pull the changes from master first? I had merged some changes into master a day or two ago, so your version didn't have those changes. Would that apply?

**Comment by kulicuu** [comment]
Yeah something like that.  I definitely screwed up the conflict resolution in my impatience.  Not because I changed anything (I didn't) , just because the delta/diff makes it look like I changed _everything_.  Again, super apologies.  I think next time I may do the pull with a `force` flag or something, because changes coming in from master wouldn't generally be changing the folder I'm working in.  
(p.s.  of course even if you hadn't had local copy everything would be retrievable by checking out a prior commit.  Just an inconvenience.)

**Comment by arxpoetica** [comment]
No worries. Just sorting it out. ;)

**Pull request merged**